src/isar_turtlebot/ros_bridge/topic.py

```diff
@@ -1,11 +1,9 @@
-import base64
 import logging
 import time
 from abc import ABC, abstractmethod
 from logging import Logger
-from pathlib import Path
 from typing import Any, Optional
-from uuid import UUID, uuid4
+from uuid import UUID
```

Comment:
```suggestion
```
This type hinting can be removed along with the `inspection_id` parameter in `ImageTopicInterface`, as it is no longer in use.